cfme/tests/storage/test_volume_snapshot.py

```diff
@@ -24,16 +26,34 @@
 STORAGE_SIZE = 1
 
 
+@pytest.fixture(params=["from_manager", "from_volume"])
+def from_manager(request):
```

Comment:
**Question** Why not paramterize the test on this? A fixture should be used test setup

something like:

```
@pytest.mark.parametrize('snapshot_create_from', [True, False], ids=['from_manager', 'from_volume'])
def test_storage_snapshot_create_cancelled_validation(volume, from_manager):
```
I see you are trying to reduce code duplication, awesome. But for readability, I would prefer it to be marked for each test